e2e/broker.go

```diff
@@ -2,17 +2,18 @@ package e2e
 
 import (
 	"encoding/json"
-	"fmt"
-
 	v3 "github.com/mysteriumnetwork/discovery/proposal/v3"
```

Comment:
Node's using `v4`. Makes sense to use it as well

Reply:
Maybe. Not ready to switch now. In scope of other task